dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/net/RumRequestInterceptor.kt

```diff
@@ -36,9 +38,10 @@ internal class RumRequestInterceptor : RequestInterceptor {
     }
 
     override fun handleResponse(request: Request, response: Response) {
+        val requestId = identifyRequest(request)
```

Comment:
nice ...this solves also our problem regarding the unique keys for resources.

Reply:
Indeed :)